packages/astro-md/package.json

```diff
@@ -11,6 +11,6 @@
   "license": "MIT",
   "dependencies": {
     "@astrojs/markdown-remark": "catalog:",
-    "gray-matter": "^4.0.3"
+    "remark-frontmatter": "^5.0.0"
```

Comment:
`gray-matter` includes eval so I wanted to just drop it. It's used transitively though, so the point is moot. Regardless, this works better.